smartseq2_single_sample/check_reference/tasks/star.wdl

```diff
@@ -0,0 +1,50 @@
+task Star {
+  File input_fastq_read1
+  File input_fastq_read2
+  File gtf
+  File star_genome
+  String sample_tag
+  String pu_tag
+  String id_tag
+  String lib_tag
+
+
+  command {
```

Comment:
In some tasks you leave spaces between command output and runtime and in others you don't. 

... I like the spaces. :-)

Reply:
I don't mind to add spaces/lines